.github/workflows/nightly_release.yml

```diff
@@ -70,12 +81,18 @@ jobs:
         run: |-
           npm run bump-version:commerce-sdk-react -- "${{ env.commerce_sdk_react_version_base }}-nightly-${{ env.release_timestamp }}"
 
-      - name: Push branch to origin (Publish to NPM)
+      - name: Push version changes to origin
         run: |-
-          git config --global user.name ${{ secrets.GIT_CONFIG_USERNAME }}
-          git config --global user.email ${{ secrets.GIT_CONFIG_EMAIL }}
           git commit -am "Release ${{ env.nightly_version }}"
-          git push --set-upstream origin release-nightly-${{ env.monorepo_version_major }}.x
+          git push --set-upstream origin nightly-releases
+
+      - name: Check Repository Clean
```

Comment:
Github will not allow triggering another workflow from branches or push events created by a different workflow. It requires creating a personal access token with selective permissions to be able to do that. This is kinda of a security issue.

Also, 
the original plan was to push changes to a `release-*` branch and let the main CI workflow take care of the release. The main CI workflow runs tests which are flaky and we'll end up with nightly releases never getting published unless someone manually re-runs the failed workflows every morning. 

So we moved the `publish-to-npm` step as a part of the original workflow to avoid issues mentioned above.